web/src/timePosted.js

```diff
@@ -1,15 +1,13 @@
 export default (postedAtSeconds) => {
+  const minute = 60, hour = 3600, day = 86400, week = 604800;
```

Comment:
I'd keep the original spelled out calculations, e.g. `3600 * 24` instead of `86400` for clarity. JS interpreters should be smart enough to optimize those as constants